src/vm.sh

```diff
@@ -109,14 +109,58 @@ function vm_up()
   cmd_manager "$flag" "$cmd"
 }
 
+function parse_vm_options()
+{
+  local long_options='help,mount,umount,up'
+  local short_options='h,mo,um,u'
```

Comment:
You will probably have to refactor these options to deal with the particular options of mount umount and up